src/components/grid-aware/HomePageLargeParagraph/HomePageLargeParagraph.module.css

```diff
@@ -0,0 +1,65 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  justify-items: center;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+  padding-bottom: 120px;
+  padding-top: 120px;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+  text-align: center;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  grid-template-rows: repeat(2, auto);
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+  }
+
+  .description {
+    margin-top: 40px;
+  }
+}
+
+.textWrapper {
+  grid-column-end: 12;
+  grid-column-start: 2;
+}
+
+.title {
+  font: var(--font-headline);
+}
+
+.description {
+  font: var(--font-title);
+  margin-top: 40px;
+
+}
+
+div p {
+  margin-top: 0;
+  padding: 0;
+}
```

Comment:
If p tags were deleted above, this shouldn't be here.

Reply:
👍 Deleted.